params/config.go

```diff
@@ -555,19 +545,7 @@ func (c *ChainConfig) String() string {
 	default:
 		engine = "unknown"
 	}
-
-	// TODO Covalent: Refactor to more generic approach and potentially introduce tag for "ecosystem" field (Ethereum, BSC, etc.)
-	if c.Consensus == ParliaConsensus {
```

Comment:
Can we keep this, but perhaps with a different condition, to print the BSC fork numbers?